src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -1045,21 +1045,25 @@ def error_raiser(msg, exception):
         )
         return query_compiler
 
-    @base_not_implemented()
     def asfreq(
```

Comment:
can you also update the docstring for this API?

Reply:
Ah forgot, will do.